api_blueprint.apib

```diff
@@ -296,3 +296,124 @@ Execute a search
             X-Domino-Api-Key: YOUR_DOMINO_API_KEY
 
 + Response 200 (application/json)
+
+## Group Models
+
+These APIs allow you to interact with your project's models.
+
+### Get models [GET /v1/projects/{username}/{project_name}/models]
+
+Get a list of models for a project.
+
++ Parameters
+    + username: `nick` (string, required) - The project owner's username.
+    + project_name: `winequality` (string, required) - The project name.
+
++ Request
+
+    + Headers
+
+            X-Domino-Api-Key: YOUR_DOMINO_API_KEY
+
++ Response 200 (application/json)
+
+    + Attributes
+
+        + objectType: `list` (string) - The type of object the API returned.
+        + data (array) - Array of models.
+            - (object)
+                + id: `57dafe43317467e7b0ef5276` (string) - The unique ID for the run.
+                + name: `Wine Quality Model` (string) - The model name.
+
+### Create a model [POST /v1/models]
```

Comment:
Publish instead of create

Reply:
Well, the endpoint itself actually hits the create function, which then calls publish. So, I think this is actually technically correct.

Reply:
Eh I think switching it to publish is fair. I initially was trying to keep the terminology closer to CRUD than to Domino's own terminology, but it looks like that goes against what's already present in the docs.